include/zephyr/xen/dom0/domctl.h

```diff
@@ -30,7 +30,7 @@ int xen_domctl_assign_dt_device(int domid, char *dtdev_path);
 int xen_domctl_bind_pt_irq(int domid, uint32_t machine_irq, uint8_t irq_type, uint8_t bus,
 			   uint8_t device, uint8_t intx, uint8_t isa_irq, uint16_t spi);
 int xen_domctl_max_vcpus(int domid, int max_vcpus);
-int xen_domctl_createdomain(int domid, struct xen_domctl_createdomain *config);
+int xen_domctl_createdomain(int *domid, struct xen_domctl_createdomain *config);
```

Comment:
Please add at least minimal doxygen doc for this public API
Any Zephyr API can be used as an example.

Reply:
Again I think we should do it all at once as a separate task. Documenting one function in the set of 20 other undocumented ones seems strange to me, and documenting all of them is way out of the scope of this task.

Reply:
Without docs NACK from my side

Reply:
I created separate issue in scope of zephyr-xenlib stabilization #97.